server/db/level/jobs.coffee

```diff
@@ -8,9 +8,9 @@ module.exports =
   # taking two arguments; a payload and an error logger function
   initQueue: (jobName, worker, maxConcurrency)->
     db = levelBase.sub "job:#{jobName}"
-
-    # If runJobsInQueue is true, push and run jobs
-    if CONFIG.runJobsInQueue
+    # Only starts job marked in runJobsInQueue, push and run jobs
+    jobKey = jobName.split(':')[1]
```

Comment:
using the full `jobName` as key seems less hazardous, as it let's more freedom in the `jobName` definition

Reply:
its prefixyied (jobname  = `wd:popularity`) 
do we need a key without a `:` in the middle for config files ?

Reply:
no, it's ok to have a `:` in the key, it just requires to use single quotes around the key:

:negative_squared_cross_mark: crashes:
```coffee
runJobsInQueue:
  wd:popularity: true
```
:white_check_mark: works:
```coffee
runJobsInQueue:
  'wd:popularity': true
```